dtfs-central-api/api-tests/v1/bank/banks-get.api-test.js

```diff
@@ -21,14 +21,8 @@ describe('/v1/bank', () => {
       expect(status).toEqual(200);
 
       const expectedBody = [
-        {
-          _id: createdBarclaysBank._id,
-          ...BANKS.BARCLAYS,
-        },
-        {
-          _id: createdHsbcBank._id,
-          ...BANKS.HSBC,
-        },
+        { ...BANKS.BARCLAYS, _id: createdBarclaysBank._id },
+        { ...BANKS.HSBC, _id: createdHsbcBank._id },
```

Comment:
I'm not sure how I feel about this change. Whenever you're in MongoDB the `_id` field comes first, so I do think the previous way this was written is easier to follow. My main concern is if someone copies this layout in the future they might add more than two fields in the single line which will make it very hard to read (I only say this because I could maybe see myself doing that if I were tired xD)